### PR TITLE
Ensure jQuery.ajax pending request detection works with Ember 3.27 under Embroider

### PR DESCRIPTION
**Pull request by stefanpenner**
This both tidies up the code, but also ensures embroider compatibility for those users with ember ember-source@3.27.x but still have jQuery.

### CODE REVIEW

File: addon-test-support/@ember/test-helpers/settled.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@
 import { _backburner } from '@ember/runloop';
 import Ember from 'ember';
 
+import EmberApplication from '@ember/application';
```
Comment (rwjblue) [inline addon-test-support/@ember/test-helpers/settled.ts:5]: I think this needs to be:

```suggestion
import EmberApplication from '@ember/application/instance';
```

**Review by rwjblue — APPROVED** [review]

**Comment by stefanpenner** [comment]
this isn't ready yet...

**Comment by stefanpenner** [comment]
it is working locally, will look at the tests.

**Comment by stefanpenner** [comment]
ok stuff is ok now.

**Comment by stefanpenner** [comment]
@rwjblue i don’t have commit here, so I’m relying on you (or someone else) to merge and release.

**Comment by rwjblue** [comment]
> i don’t have commit here

😱 I have fixed this bug

**Review by rwjblue — APPROVED** [review]

**Pull request merged**